StardewXnbHack/Framework/PlatformExtensions.cs

```diff
@@ -0,0 +1,15 @@
+using StardewModdingAPI.Toolkit.Utilities;
+
+namespace StardewXnbHack.Framework
+{
+    internal static class PlatformExtensions
+    {
+        /// <summary>Get whether the platform uses Mono.</summary>
+        /// <param name="platform">The current platform.</param>
+        public static bool IsMono(this Platform platform)
+        {
+            string platformAsString = platform.ToString();
+            return platformAsString == nameof(Platform.Linux) || platformAsString == nameof(Platform.Mac);
```

Comment:
We can simplify this a bit:
```suggestion
            return platform == Platform.Linux || platform == Platform.Mac;
```